hdl/modules/fofb_sys_id/cheby/wb_fofb_sys_id_regs.cheby

```diff
@@ -28,3 +28,153 @@ memory-map:
+                      - field:
+                          name: en_distort
+                          range: 0
+                          description: |
+                            Enables/disables PRBS-based distortion on BPM
+                            positions
+
+                            write 0: distortion disabled
+                            write 1: distortion enabled
+                - memory:
+                    name: distort_ram
+                    memsize: 2k
+                    address: next
+                    description: Distortion levels RAM
+                    children:
+                      - reg:
+                          name: levels
+                          width: 32
+                          access: rw
+                          description: |
+                            Two S16FP0-bit distortion levels, one for each PRBS
+                            value.
+
+                            15 - 0: distortion level for PRBS value 0
+                            31 - 16: distortion level for PRBS value 1
+                          children:
+                            - field:
+                                name: level_0
+                                range: 15-0
+                                description: |
+                                  Distortion level for PRBS value 0 in
+                                  nanometers
+                            - field:
+                                name: level_1
+                                range: 31-16
+                                description: |
+                                  Distortion level for PRBS value 1 in
+                                  nanometers
+          - block:
+              name: sp_distort
+              description: Interface to prbs_sp_distort regs
+              address: next
+              children:
+                - reg:
+                    name: ctl
+                    width: 32
+                    access: rw
+                    address: next
+                    description: |
+                      PRBS-based distortion for setpoints control register
+                    children:
+                      - field:
+                          name: en_distort
+                          range: 0
+                          description: |
+                            Enables/disables PRBS-based distortion on setpoints
+
+                            write 0: distortion disabled
+                            write 1: distortion enabled
+                - repeat:
+                    name: ch
+                    count: 12
+                    description: |
+                      Distortion levels registers (one for each channel)
+                    children:
+                      - reg:
+                          name: levels
+                          width: 32
+                          access: rw
+                          description: |
+                            Two S16FP0-bit distortion levels, one for each PRBS
```

Comment:
I find using this `S16FP0` notation confusing here when you can simply write 'Two signed 16 bits...'